awscli/customizations/ecs/deploy.py

```diff
@@ -202,9 +204,16 @@ def _validate_code_deploy_resources(self, client):
         validator = CodeDeployValidator(client, self.resources)
         validator.describe_cd_resources()
         validator.validate_all()
+        return validator.get_deployment_wait_time()
 
 
 class CodeDeployer():
+
+    MSG_WAITING = ("Waiting for {deployment_id} to "
+                   "succeed{custom_wait_msg}...\n")
+
+    MSG_CUSTOM_WAIT = " (will timeout after {wait} minutes)"
```

Comment:
Is there any reason to not combine these messages and just print the amount of time to be waited every time?

Reply:
Original though was it may be helpful to be visually distinct from the default behavior, which is currently 30 min (vs. configured min + 10). However, I don't really see a downside to showing it all the time, and that would decrease the # of code paths we need to maintain & test. Will make that change.